{
  "id": "CVE-2024-24780",
  "sourceIdentifier": "security@apache.org",
  "published": "2025-05-14T11:15:47.683",
  "lastModified": "2025-05-14T11:15:47.683",
  "vulnStatus": "Received",
  "descriptions": [
    {
      "lang": "en",
      "value": "Remote Code Execution with untrusted URI of UDF vulnerability in Apache IoTDB. The attacker who has privilege to create UDF can register malicious function from untrusted URI.\n\nThis issue affects Apache IoTDB: from 1.0.0 before 1.3.4.\n\nUsers are recommended to upgrade to version 1.3.4, which fixes the issue."
    }
  ],
  "metrics": {},
  "references": [
    {
      "url": "https://lists.apache.org/thread/xphtm98v3zsk9vlpfh481m1ry2ctxvmj",
      "source": "security@apache.org"
    },
    {
      "url": "http://www.openwall.com/lists/oss-security/2025/05/14/2",
      "source": "af854a3a-2127-422b-91ae-364da2661108"
    }
  ]
}